plugins/org.yakindu.sct.ui.editor/src/org/yakindu/sct/ui/editor/editor/StatechartDiagramActionbarContributor.java

```diff
@@ -26,15 +36,61 @@
 	@Override
 	public void init(IActionBars bars) {
 		super.init(bars);
+
 		// workaround for https://bugs.eclipse.org/bugs/show_bug.cgi?id=346648
 		bars.setGlobalActionHandler(GlobalActionId.SAVE, null);
 		bars.getToolBarManager().add(new DocumentationMenuAction());
-		
+
+		removeAppearanceButtonsInSimulationAndTestingPerspective(bars);

```

Comment:
Rename to something more readable, e.g. `showButtonsBasedOnPerspective` or even shorter `showButtonsByPerspective`